lms/djangoapps/bulk_email/models.py

```diff
@@ -470,6 +470,27 @@ def __str__(self):
             not_en = ""
         return f"Course '{str(self.course_id)}': Instructor Email {not_en}Enabled"
 
+class DisabledCourse(models.Model):
+    """
+    Disable the bulk email feature for specific courses.
+
+    .. no_pii:
+    """
+
+    course_id = CourseKeyField(max_length=255, db_index=True, unique=True)
+    email_disabled = models.BooleanField(default=False)
```

Comment:
Do we really need this?

Reply:
Yeah Good point. We don't want courses on the list with this disabled.